waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/impl/PrefixBasedDatabaseMappingService.java

```diff
@@ -430,6 +437,7 @@ public void close() throws IOException {
       for (MetaStoreMapping metaStoreMapping : mappingsByPrefix.values()) {
         metaStoreMapping.close();
       }
+      TokenWrappingHMSHandler.removeToken();
```

Comment:
this line should go: it's untestable and there is lifecycle where this is calls via the `TTransportMonitor` which call closes on the TokenWrappingHMSHandler which should remove the token already.